engines/bastion_katello/app/assets/javascripts/bastion_katello/organizations/fenced-pages.service.js

```diff
@@ -42,8 +45,14 @@ angular.module('Bastion.organizations').service('FencedPages', ['$state',
         };
 
         this.isFenced = function (toState) {
-            var stateUrl = $state.href(toState);
-            return fencedPages.indexOf(getRootPath(stateUrl)) !== -1;
+            var stateUrl = $state.href(toState),
+            rootPath;
```

Comment:
Declaring multiple variables in a single `var` statement is fine, but it's confusing if it's on two lines.  I would just say
```js
var rootPath;
```
instead

Reply:
updated